plugins/webhooks/index.js

```diff
@@ -634,53 +630,64 @@ async function createEvents(eventFactory, userFactory, pipelineFactory,
     const events = [];
     const meta = createMeta(parsed);
 
-    for (let i = 0; i < pipelines.length; i += 1) {
-        const p = pipelines[i];
-        /* eslint-disable no-await-in-loop */
-        const pipelineBranch = await p.branch;
-        /* eslint-enable no-await-in-loop */
-        const startFrom = determineStartFrom(action, type, branch, pipelineBranch);
+    const pipelineTaple = await Promise.all(pipelines.map(async (p) => {
```

Comment:
what is a taple? 🤔

Reply:
It was misspeling of `tuple`.
`tuple` means just a combination in this codes.